server/src/main/java/com/paypal/selion/utils/ConfigParser.java

```diff
@@ -48,6 +48,10 @@
      *         represented by the JSON file passed via the JVM argument <b>SeLionConfig</b>
      */
     public static ConfigParser parse() {
+        LOGGER.entering();
+        if (parser == null) {
```

Comment:
@sundaramrajendran Will you please elaborate on the issue this change is fixing? The code change you made here is a revert of a previous change you made. See https://github.com/sundaramrajendran/SeLion/commit/b8064e0686225b38982510fe1c7f0c72748dcb7c#diff-f0886e3373d41f8f4e2ca67ec83de2e4